MyHashMap.java

```diff
@@ -0,0 +1,77 @@
+// Time Complexity : O(1) //amortized
+// Space Complexity :
```

Comment:
O(n) where n is the length of the LL formed at any index of the array.
O(10000) for array * O(n) for LL = O(n).